src/core/dom/image/const.ts

```diff
@@ -0,0 +1,53 @@
+/*!
+ * V4Fire Client Core
+ * https://github.com/V4Fire/Client
+ *
+ * Released under the MIT license
+ * https://github.com/V4Fire/Client/blob/master/LICENSE
+ */
+
+import { DefaultParams } from 'core/dom/image';
+
+/**
+ * This variable is used to override the default (`defaultBrokenImageOptions`, `defaultPreviewImageOptions`) values.
```

Comment:
Такие подробности избыточны, просто

```suggestion
 * Default parameters for image placeholders: `defaultBrokenImageOptions` and `defaultPreviewImageOptions`
```